src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -1782,7 +1782,7 @@ public static ActivityTraceId CreateFromBytes(ReadOnlySpan<byte> idData)
             if (idData.Length != 16)
                 throw new ArgumentOutOfRangeException(nameof(idData));
 
-            return new ActivityTraceId(HexConverter.ToString(idData, HexConverter.Casing.Lower));
+            return new ActivityTraceId(Convert.ToHexStringLower(idData));
```

Comment:
> error CS0117: 'Convert' does not contain a definition for 'ToHexStringLower'

Ah, looks this project builds for netstandard, .Netframework and previous .NET core releases either need to #if-def to only build on Core current release, or maybe bettor to revert changes.

Reply:
> > error CS0117: 'Convert' does not contain a definition for 'ToHexStringLower'
> 
> Ah, looks this project builds for netstandard, .Netframework and previous .NET core releases either need to #if-def to only build on Core current release, or maybe bettor to revert changes.

Should I use `#if NET9_0_OR_GREATER`?

Reply:
Yep, thanks